libs/pika/async_cuda_base/include/pika/async_cuda_base/cublas_handle.hpp

```diff
@@ -27,19 +38,41 @@ namespace pika::cuda::experimental {
         static PIKA_EXPORT cublasHandle_t create_handle(int device, whip::stream_t stream);
 
     public:
+        /// \brief Constructs a new cuBLAS handle with the default stream.
         PIKA_EXPORT cublas_handle();
+
+        /// \brief Constructs a new cuBLAS handle with the given stream.
         PIKA_EXPORT explicit cublas_handle(cuda_stream const& stream);
         PIKA_EXPORT ~cublas_handle();
         PIKA_EXPORT cublas_handle(cublas_handle&&) noexcept;
         PIKA_EXPORT cublas_handle& operator=(cublas_handle&&) noexcept;
         PIKA_EXPORT cublas_handle(cublas_handle const&);
         PIKA_EXPORT cublas_handle& operator=(cublas_handle const&);
 
+        /// \brief Check if the handle is valid.
+        ///
+        /// \return true if the pool refers to a valid handle, false otherwise (e.g. if the handle
```

Comment:
?

```suggestion
        /// \return true if the handle refers to a valid handle, false otherwise (e.g. if the handle
```

Reply:
Thanks for the detailed look! Fixed in the latest push.